src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator, PerimeterCalculator {
```

Comment:
Make this class as an abstract and you will not be needed to declare (Override) these methods:
```
public void draw() {
    }

    public double getArea() {
        return 0;
    }

    public double getPerimeter() {
        return 0;
    }
```